scripts/ja/verify-spelling.sh

```diff
@@ -26,6 +26,9 @@ TARGET_WORDS="${TARGET_WORDS} アーキテクチャー:アーキテクチャ"
 TARGET_WORDS="${TARGET_WORDS} コントローラ[^ー]:コントローラー"
 TARGET_WORDS="${TARGET_WORDS} オペレータ[^ー]:オペレーター"
 TARGET_WORDS="${TARGET_WORDS} パラメータ[^ー]:パラメーター"
+TARGET_WORDS="${TARGET_WORDS} レイヤ[^ー]:レイヤー"
+TARGET_WORDS="${TARGET_WORDS} クラスタ[^ーリ]:クラスター" # 例外的にクラスタリングはクラスターリングにしない
```

Comment:
@nasa9084 @t-inu 

I have two items to think.
1. `クラスタリング` is valid for localization? or `クラスターリング` is better?
2. I should write this comment(# 例外的にクラスタ...) in English? I think Japanese comment is OK because that this script is for Japanese.

Reply:
1. I vote for `クラスタリング`.
2. I think Japanese comment is OK.

Reply:
Thank you for comments!!
I can move forward.